### ISSUE TITLE
How to prevent body scroll when modal is opened ON TOUCH DEVICES?

### ISSUE THREAD

**Issue by 7iomka** [fancyapps/fancybox#2189]
**Describe the bug**
In a new (latest 3.5.x) version by default when I touch modal content (if touch: false) it is possible to see - body scrolling. In previous version (I tested v3.3.5) everything was fine 

**To Reproduce**
1. Go to provide demo with latest version, change in inspector (i like Chrome) mobile device, reload page
2. scroll (swipe) to bottom, will open modal programmatically
3. try to swipe content(!) of modal
4. See body scrolling

**Expected behavior**
Body scrolling is prevented by default for opened modals when user trying to swipe modal content (it makes no difference whether it exceeds the screen boundaries or not)

**Screenshots**
1) DEMO with latest version: 
Codepen: https://codepen.io/7iomka/full/Nojvwr
Video: https://cl.ly/b69bc5ff9afd

2) DEMO with version 3.3.5 (works as expected)
Codepen: https://codepen.io/7iomka/full/BMRdYj
Video: https://cl.ly/1b67c046e8f9

**Desktop (please complete the following information):**
 - OS: Windows 10 (x64)
 - Browser Chrome
 - Version latest

**Additional context**
I initially chose your plugin because it solves the scroll problem, unlike many alternatives (not to mention the unrealistically cool work you have done towards creating a universal plug-in for galleries with different types of content), so I am very surprised that I discovered behavior in the latest version of the plugin (which, by the way, it was installed only yesterday).
If there is some documented or undocumented setting that can fix this behavior, I would appreciate it if you tell me!
Thanks!

**Comment by fancyapps** [comment]
TL;DR This is how mobile browsers work and there is nothing you can do about that until this webkit bug is fixed - https://bugs.webkit.org/show_bug.cgi?id=153852

A bit of explaining why this seemed to work in previous versions. Not so long time ago, there were multiple issue reports about "input field focus issue", see #1646 That issue was caused by another webkit bug and the only workaround was to set `position:fixed` for `body` element. This hack had two side-effects:
1) it also fixed background scrolling issue (the problem that caused you to create this post);
2) performance issues; broken layout-dependent scripts; position change causes address bar to focus and therefore page jumps; and possibly other issues;

After some time, that webkit bug was fixed and there was no need for that hack, therefore it was removed. But, there is no other solution to prevent page scrolling, but, if you wish, you can add the same hack by yourself:

CSS
```CSS
body.fancybox-iosfix {
	position: fixed;
	left: 0;
	right: 0;
}
```

JS
```JS
$('[data-fancybox="testemptycaption"]').fancybox({
    onInit: function () {
        if ($.fancybox.isMobile) {
            $('body').css('top', $('body').scrollTop() * -1).addClass('fancybox-iosfix');
        }
    },
    afterClose: function () {
        var offset;

        if ($('body').hasClass('fancybox-iosfix')) {
            offset = parseInt(document.body.style.top, 10);

            $('body').removeClass('fancybox-iosfix').css('top', '').scrollTop(offset * -1);
        }
    }
});
```
